pallets/ajuna-board/src/lib.rs

```diff
@@ -157,10 +159,15 @@ pub mod pallet {
 			// There is potentially more than one attack vector here as anyone could assign any
 			// account to a board and hence block them from playing in a legitimate game
 			// As this would be ran in L2 we may want to check that we are in L2??
+			//
+			// One way to solve it would be not "adding" a player to a game but "inviting" them.
+			// After being invited, a player (AccountId) would be able to accept or decline it.
 			let _ = ensure_signed(origin)?;
 
 			// Ensure we have players
 			ensure!(!players.is_empty(), Error::<T>::NotEnoughPlayers);
+			// Maybe using a global board counter here, incremented (+1) on every new_game().
+			// That way we'd avoid game creators being aware of available "board_ids".
```

Comment:
Yeah good point.
We probably need to change `BoardStates` to be a double map of `GameCreator, BoardId => BoardState` so the game creators are aware only of what they created.